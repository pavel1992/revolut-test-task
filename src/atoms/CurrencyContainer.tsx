import styled from 'styled-components';

import { colors } from "../colors";

export interface CurrencyContainerProps {
  isFromCurrency?: boolean;
}

export const CurrencyContainer = styled.div<CurrencyContainerProps>`
  background-color: ${(props: CurrencyContainerProps) => props.isFromCurrency ? colors.lightBlue : colors.darkBlue};
  height: 100%;
  display: flex;
  flex-direction: column;
  position: relative;
`;