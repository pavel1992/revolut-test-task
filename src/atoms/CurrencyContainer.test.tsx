import * as React from 'react';
import renderer from 'react-test-renderer';

import { CurrencyContainer } from "./CurrencyContainer";


test('renders currency container correctly', () => {
  const renderedApp = renderer
    .create(<CurrencyContainer/>)
    .toJSON();
  expect(renderedApp).toMatchSnapshot();
});
