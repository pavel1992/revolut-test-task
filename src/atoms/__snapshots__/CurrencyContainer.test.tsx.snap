// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders currency container correctly 1`] = `
<div
  className="sc-AxjAm cWWsEF"
/>
`;
